recipes/newrelic/apm/dotNet/windows/net-install.yml

```diff
@@ -226,14 +242,20 @@ install:
           }
 
           Write-Host "Sending requests to your applications..."
-          if ($NEW_RELIC_IIS_URL) {
-            Write-Host $NEW_RELIC_IIS_URL
-            while($val -ne 3) {
-              $val++
-              Invoke-WebRequest -UseBasicParsing -Method Get -TimeoutSec 5 -Uri $NEW_RELIC_IIS_URL -ErrorAction SilentlyContinue | Out-Null
+          try {
```

Comment:
@jaffinito should we still try up to 3 times when it failed? It's most likely a timeout because the app doesn't start in 5 second, it might be good to try again?

Reply:
I guess its a tradeoff.  If we check inside the while, we will always try 3 times, even if there are errors, but we take care of the slow start up problem.  If we check outside, we will only check once on errors, but miss the slow start up.  I was targeting the second use case initially, but I think give our goals with this work, the first case is the one to solve for.  I will put in a change to address it.